exercises/practice/kindergarten-garden/tests/kindergarten_garden.cairo

```diff
@@ -0,0 +1,170 @@
+use kindergarten_garden::plants;
+
+#[test]
+fn garden_with_single_student() {
+    let diagram = "RC
+GG";
+    let student = "Alice";
+    let expected = array!["radishes", "clover", "grass", "grass"];
+    assert_eq!(plants(diagram, student), expected);
+}
+
+#[test]
+#[ignore]
+fn different_garden_with_single_student() {
+    let diagram = "VC
+RC";
+    let student = "Alice";
+    let expected = array!["violets", "clover", "radishes", "clover"];
+    assert_eq!(plants(diagram, student), expected);
+}
+
+#[test]
+#[ignore]
+fn garden_with_two_students() {
+    let diagram = "VVCG
+VVRC";
+    let student = "Bob";
+    let expected = array!["clover", "grass", "radishes", "clover"];
+    assert_eq!(plants(diagram, student), expected);
+}
+
+#[test]
+#[ignore]
+fn second_students_garden() {
```

Comment:
This test name is pretty similar to the one before it. To avoid confusion, I'd consider perhaps `fn multiple_students_bobs_garden () {`. The next test can then be `fn multiple_students_charlies_garden () {`.

Reply:
I usually name tests after canonical data's description. For these one the descriptions were respectively `"second student's garden"` and `"third student's garden"`, see [canonical-data.json](https://github.com/exercism/problem-specifications/blob/main/exercises/kindergarten-garden/canonical-data.json#L42)